### DISCUSSION TITLE
User accounts assigned to site

### DISCUSSION THREAD

**Discussion by samuelbirch** [craftcms/cms#8734] [Ideas]
It would be good to be able to have user accounts assigned to a site. So a client can know which site on a multi-site setup the user belongs to.

This also leads on to allowing a user to be able to have a seperate account for each site, with the same email address. This would be especially useful for a multi-site e-commerce setup, to stop users getting confused that an account already exists for a site they have never visited.

**Comment by lukeholder** [comment]
Moving this to craftcms/cms
